themes/jquery/menu-header.php

```diff
@@ -64,7 +64,7 @@ function menu_header_jqueryui_com() {
 
 function menu_header_jquery_org() {
 	return array(
-		'https://jquery.org/' => 'Home',
+		'https://jquery.com/' => 'Home',
```

Comment:
Neither org nor home make sense to me here... jquery.org no longer has a home page, and jquery.com is a different site. I'd rather remove this link altogether.